### PR TITLE
add troubleshooting topic

### LINKED ISSUE #409: Document how to connect to a live site via SFTP
Story:
As a Pantheon User I want connect to my live site via SFTP so that I don't see errors being bounced between servers. 

When users connect with some FTP clients:

FileZilla
Yummy
CyberDuck
They are using the application's default settings and for Pantheon that can cause some issues. They need to set their connection limit to 1 and then connect, that way they will not get errors while on their live site.

Screenshots of where the settings are for each client:

Yummy FTP Client: https://www.evernote.com/shard/s203/sh/c58733b1-b259-429a-ad5d-71eb9d05b3e7/ce06c2c099616b536043807492ee5dfc
FileZilla: https://www.evernote.com/shard/s203/sh/f1fa6054-c557-436e-be31-53c7a74981bc/fba13129d7c06468462f918d1a0291bd
CyberDuck: does not have a setting I can find

submitted by @atjuch

### PR DESCRIPTION
**Pull request by nataliejeremy**
This issue closes #409. @suzannealdrich please review for accuracy--thanks!

### CODE REVIEW

**Comment by nataliejeremy** [comment]
@atjuch Can you review this one?

**Comment by nataliejeremy** [comment]
@atjuch Can you review and +1? Thx!

File: source/docs/articles/sites/code/developing-directly-with-sftp-mode.md [hunk 1]
```diff
@@ -104,3 +104,6 @@ SFTP mode works with any standards-compliant SFTP client, including many GUI too
 
 This is by design. Please see [Using the Pantheon Workflow
 ](/docs/articles/sites/code/using-the-pantheon-workflow#understanding-write-permissions-in-test-&-live) to learn why.
+
+###I am receiving errors connecting to my server with an SFTP client.
+This is caused by using the application's default connection settings. We recommend you set the connection limit to **1** and then connect to your site.
```
Comment (bmackinney) [inline source/docs/articles/sites/code/developing-directly-with-sftp-mode.md]: Show/explain how to do this?

**Comment by nataliejeremy** [comment]
@rachelwhitton please merge--thanks!

**Pull request merged**